client/src/components/FrontPage/CourseTable.js

```diff
@@ -0,0 +1,47 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableHead from '@mui/material/TableHead';
+import CourseTableRow from './CourseTableRow';
+import CourseHeadTableRow from './CourseHeadTableRow';
+
+//used to sort the table rows by course code
+function sortByCode(a, b) {
```

Comment:
Is sortByCode generic enough to be used with other components? If so, could be moved somewhere as a service maybe.

Reply:
Okay, working on this.